### ISSUE TITLE
Windows: `telegraf --test` sample config errors

### ISSUE THREAD

**Issue by jstirnaman** [influxdata/telegraf#13775]
### Relevant telegraf.conf

```toml
# The default sample config in the Telegraf v1.26-1.27.3 release for Windows.
```

### Logs from Telegraf

```text
PS C:\Program Files\InfluxData\telegraf> .\telegraf.exe `
>> --config "C:\Program Files\InfluxData\telegraf\telegraf.conf" --test
2023-08-14T20:53:06Z I! Loading config: C:\Program Files\InfluxData\telegraf\telegraf.conf
2023-08-14T20:53:06Z E! error loading config file C:\Program Files\InfluxData\telegraf\telegraf.conf: plugin inputs.processes: line 3828: configuration specified the fields ["use_sudo"], but they weren't used
```

Running with `--debug` had no affect on the output.
See the original reported issue https://github.com/influxdata/docs-v2/pull/4935
```


### System info

Telegraf v1.27.3, Windows 11

### Docker

_No response_

### Steps to reproduce

1. Follow the Windows installation instructions https://docs.influxdata.com/telegraf/v1.27/install/?t=Windows
2. In PowerShell, run `.\telegraf.exe `
>> --config "C:\Program Files\InfluxData\telegraf\telegraf.conf" --test`


### Expected behavior

Telegraf should start, run, and exit without errors.
After commenting out the `[[inputs.processes]]` configuration lines, it ran as expected.

### Actual behavior

```
PS C:\Program Files\InfluxData\telegraf> .\telegraf.exe `
>> --config "C:\Program Files\InfluxData\telegraf\telegraf.conf" --test
2023-08-14T20:53:06Z I! Loading config: C:\Program Files\InfluxData\telegraf\telegraf.conf
2023-08-14T20:53:06Z E! error loading config file C:\Program Files\InfluxData\telegraf\telegraf.conf: plugin inputs.processes: line 3828: configuration specified the fields ["use_sudo"], but they weren't used
```

### Additional info

_No response_

**Comment by powersj** [comment]
Fixed with #13762 and will go out on Monday

**Comment by jstirnaman** [comment]
> Fixed with #13762 and will go out on Monday

Oops, thanks. I should've looked harder.
